src/plugins/unified_histogram/public/chart/breakdown_field_selector.tsx

```diff
@@ -102,6 +102,12 @@ export const BreakdownFieldSelector = ({
           defaultMessage: 'Select breakdown field',
         }
       )}
+      optionMatcher={({ option, searchValue }) => {
+        return fieldNameWildcardMatcher(
+          { name: option.key || option.label, displayName: option.label },
+          searchValue
+        );
+      }}
```

Comment:
Hi @mbondyra,

Great idea on adding wildcard matching to other places too!

Looking at the PR I think it would be better to create a wrapper function like `comboBoxFieldOptionMatcher` (also in '@kbn/field-utils') and reuse it in these files to avoid duplication and to not create functions in react props.